pkg/sloop/store/typed/eventcounttable_test.go

```diff
@@ -126,8 +98,106 @@ func Test_EventCount_TestGetMinMaxPartitions(t *testing.T) {
 
 	assert.Nil(t, err)
 	assert.True(t, found)
-	assert.Equal(t, untyped.GetPartitionId(someTs), minPartition)
-	assert.Equal(t, untyped.GetPartitionId(someTs), maxPartition)
+	assert.Equal(t, someMinPartition, minPartition)
+	assert.Equal(t, someMaxPartition, maxPartition)
+}
+
+func Test_EventCount_GetPreviousKey_Success(t *testing.T) {
```

Comment:
can this test (and others) be moved in to the template for the generated tests? They seem **very** similar

Reply:
done